tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -192,16 +205,88 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
+
+                      goK3SAZParams = [ string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL}"),
+                                  string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN}"),
+                                  string(name: 'USER_TOKEN', value: "${USER_TOKEN}"),
+                                  string(name: 'RANCHER_SERVER_VERSION', value: "${rancher_version}"),
+                                  string(name: 'TIMEOUT', value: "${TIMEOUT}"),
+                                  text(name: 'CONFIG', value: "${provision-config.yaml}"),
+                                  string(name: 'REPO', value: "https://github.com/rancher/rancher.git"),
+                                  string(name: 'BRANCH', value: branch),
+                                  string(name: 'TEST_PACKAGE', value: "provisioning/k3s"),
+                                  string(name: 'GOTEST_TESTCASE', value: "${k3sAZTestCase}") ]                  
+
+                      goK3SCustomParams = [ string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL}"),
+                                  string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN}"),
+                                  string(name: 'USER_TOKEN', value: "${USER_TOKEN}"),
+                                  string(name: 'RANCHER_SERVER_VERSION', value: "${rancher_version}"),
+                                  string(name: 'TIMEOUT', value: "${TIMEOUT}"),
+                                  text(name: 'CONFIG', value: "${provision-config.yaml}"),
+                                  string(name: 'REPO', value: "https://github.com/rancher/rancher.git"),
+                                  string(name: 'BRANCH', value: branch),
+                                  string(name: 'TEST_PACKAGE', value: "provisioning/k3s"),
+                                  string(name: 'GOTEST_TESTCASE', value: "${k3sCustomTestCase}") ]
+
+                      // Rancher server 1 is used to test RKE1, RKE2 and K3s clusters.
                       // DO Note: https://github.com/rancher/qa-tasks/issues/318
                       // jobs["do"] = { build job: 'rancher-v3_ontag_do_certification', parameters: params }
                       jobs["ec2"] = { build job: 'rancher-v3_ontag_ec2_certification', parameters: params }
                       jobs["az"] = { build job: 'rancher-v3_ontag_az_certification', parameters: params }
                       jobs["custom"] = { build job: 'rancher-v3_ontag_custom_certification', parameters: params}
+                      jobs["go-rke2-ec2"] = { build job: 'go-automation-freeform-job', parameters: goRKE2EC2Params }
+                      jobs["go-rke2-az"] = { build job: 'go-automation-freeform-job', parameters: goRKE2AZParams }
+                      jobs["go-rke2-custom"] = { build job: 'go-automation-freeform-job', parameters: goRKE2CustomParams }
+                      jobs["go-k3s-ec2"] = { build job: 'go-automation-freeform-job', parameters: goK3SEC2Params }
```

Comment:
The reason for 2 servers was also to distribute load; depending on the outcome of the jenkins run, we should consider moving half of the new tests to server 2.

Reply:
Understood, thanks for that. I split them, 3 into each server, respectively.

Reply:
Yeah I don't want to merge anything that still needs to be tested, let's sync up so I can make temp job for you test these changes.